hexrd/fitting/fitpeak.py

```diff
@@ -77,7 +75,6 @@ def quad_fit_obj(x, a, b, c):
 
 def quad_fit_jac(x, a, b, c):
     x = np.asarray(x)
-    return a*x**2 + b*x + c
```

Comment:
Are we sure this is the incorrect return? The return that's staying doesn't seem to include `a`, `b`, or `c`.